Create ui-c8y-1019-8-0-input-range-component-improvements-5505-graft-release-cd-5569.md (#1804)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1019-8-0-input-range-component-improvements-5505-graft-release-cd-5569.md

---
date: ""

> title: Input range component improvements (#5505) [GRAFT][release/cd] (#5569)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Improvements to input range component
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57884
version: 1019.8.0
---

> Input range component improvements (#5505) [GRAFT][release/cd] (#5569)

Author: The Claude AI suggested to use the following body:
```suggestion
The input range component has been improved. Changes include fixing issues with the component not updating when the model changes, improving accessibility, fixing visual inconsistencies across browsers, and fixing a bug where the value could temporarily show as undefined after interacting with the component.
```

> The input range component (c8y-range) can be used to show a slider to configure a range. The styling and documentation for this component was improved, so that it is more suiting the overall styling of components and forms. 

Reviewer: ```suggestion
The input range component (`c8y-range`) can be used to show a slider to configure a range. The styling and documentation for this component has been improved to better suiting the overall styling of components and forms. 
```
